src/controllers/filtercontroller.cpp

```diff
@@ -190,15 +189,15 @@ void FilterController::setCurrentFilter(int attachedIndex, bool isNew)
 void FilterController::onFadeInChanged()
 {
     if (m_currentFilter) {
-        emit m_currentFilter->changed();
+        emit m_currentFilter->changed("");
```

Comment:
Please remove this and other calls with empty string, and instead add a default parameter value to the `changed` signal:
`void changed(QString name = QString());`

Reply:
done